plugins/modules/remote_management/manageiq/manageiq_policies.py

```diff
@@ -42,7 +42,11 @@
   resource_name:
     description:
       - the name of the resource to which the profile should be [un]assigned
-    required: true
+    required: only if resource_id is not set, mutually exclusive
+  resource_id:
+    description
+      - the id of the resource to which the profile should be [un]assigned
+    required: only if resource_name is not set, mutually exclusive
```

Comment:
`required` must be a boolean, not a string! Leave it away and put that string into `description`.

Reply:
Oops. Thanks for pointing that out.